examples/config/condor/50-scitokens-credmon.conf

```
@@ -4,8 +4,8 @@
 #
 # DAEMON_LIST = $(DAEMON_LIST), CREDD, SEC_CREDENTIAL_MONITOR
 
-# MANDATORY for OAuth credmon: Uncomment this lines to have the CredD
-# use the OAuth2 mode of managing credentials
+# MANDATORY for OAuth credmon on HTCondor < 8.9.2: Uncomment this
+# line to have the CredD use the OAuth2 mode of managing credentials
 #
 # CREDD_OAUTH_MODE = True
```

Can you base this on the version? e.g.
```suggestion
if version < 8.9.2
  CREDD_OAUTH_MODE = True
endif
```